packages/orbit-components/.storybook/preview.ts

```diff
@@ -29,6 +29,29 @@ const preview: Preview = {
   decorators: [orbitDecorator],
   parameters: {
     viewport: { viewports },
+    a11y: {
+      element: "#storybook-root",
```

Comment:
Actually it would be much simpler to wrap the story into a div with a `.story-content` class and set it here. So we don't need config at all :)

```diff
diff --git a/packages/orbit-components/.storybook/orbitDecorator.tsx b/packages/orbit-components/.storybook/orbitDecorator.tsx
index 5ad9607c5..34ec0a0ea 100644
--- a/packages/orbit-components/.storybook/orbitDecorator.tsx
+++ b/packages/orbit-components/.storybook/orbitDecorator.tsx
@@ -51,7 +51,7 @@ const OrbitDecorator: Decorator = (storyFn, context) => {
         <Text spaceAfter="largest" type={inverted ? "white" : "primary"}>
           {context.parameters?.info}
         </Text>
-        {children}
+        <div className="story-content">{children}</div>
         <div dir="ltr" className="mt-600 relative">
           <Tile>
             <Highlight theme={themes.vsLight} code={code} language="tsx">
diff --git a/packages/orbit-components/.storybook/preview.ts b/packages/orbit-components/.storybook/preview.ts
index 5441c320b..b7362c19d 100644
--- a/packages/orbit-components/.storybook/preview.ts
+++ b/packages/orbit-components/.storybook/preview.ts
@@ -30,27 +30,7 @@ const preview: Preview = {
   parameters: {
     viewport: { viewports },
     a11y: {
-      element: "#storybook-root",
-      config: {
-        rules: [
-          {
-            id: "aria-required-attr",
-            selector: '*:not(div.text-heading-title0[role="heading"])',
-          },
-          {
-            id: "color-contrast",
-            selector: "*:not(pre .attr-name.tag.token)",
-          },
-          {
-            id: "color-contrast-enhanced",
-            selector: "*:not(pre .attr-name.tag.token)",
-          },
-          {
-            id: "scrollable-region-focusable",
-            selector: "*not(pre.overflow-x-auto)",
-          },
-        ],
-      },
+      element: ".story-content",
     },
   },
 };
```

Reply:
yes, that's also a good idea to include only wrapped `children` prop. thx. 👍

Reply:
Perfect, that's all we need for a separate PR 😄